appveyor.yml

```diff
@@ -24,6 +28,9 @@ install:
   - SET PYTHONPATH=%PYTHONHOME%
   - SET PATH=%PYTHONHOME%\Scripts;%PYTHONHOME%;%PATH%
 
+  - ECHO Turning off firewall...
+  - ps: Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
```

Comment:
That's probably not necessary. I added it to try and figure out why I was having issues with the updated build process. Turned out to be slow starts in RabbitMQ and `epmd`.

Reply:
Just to be clear regarding my feedback items asking for comments to be added, I meant adding the comments in the code so that other maintainers and reviewers wouldn't be wondering why we're doing these things.

Reply:
@lukebakken: Just to be clear regarding my feedback items asking for comments to be added, I meant adding the comments in the code so that other maintainers and reviewers wouldn't be wondering why we're doing these things.

Reply:
Oh, I meant that I'm going to remove `Set-NetFirewallProfile` from the build :smile: